distribution/docs/src/test/java/alliance/docs/DocumentationTest.java

```diff
@@ -0,0 +1,55 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package alliance.docs;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.stream.Stream;
+import org.junit.Test;
+
+public class DocumentationTest {
+
+  private static final String UNRESOLVED_DIRECTORY_MSG = "Unresolved directive";
+
+  private static final String DOCS_DIRECTORY = "docs";
+
+  private static final String HTML_DIRECTORY = "html";
+
+  @Test
+  public void testDocumentationIncludes() throws IOException, URISyntaxException {
+
+    Path testPath = Paths.get(this.getClass().getResource("").toURI());
+    Path targetDirectory = testPath.getParent().getParent().getParent();
+    Path docPath =
```

Comment:
✏️ You could pass a maven property as a system property that points to the path of html directory through the surefire plugin instead of traversing the path. http://maven.apache.org/surefire/maven-surefire-plugin/examples/system-properties.html